tracim/tracim/lib/email_fetcher.py

```diff
@@ -185,11 +198,12 @@ def run(self) -> None:
             time.sleep(self.delay)
             try:
                 self._connect()
-                messages = self._fetch()
-                # TODO - G.M -  2017-11-22 retry sending unsended mail
-                # These mails are return by _notify_tracim, flag them with "unseen" # nopep8
-                # or store them until new _notify_tracim call
-                cleaned_mails = [DecodedMail(msg) for msg in messages]
+                with self.lock.acquire(
```

Comment:
What appens if lock acquire fail ? filelock raise something ?

Reply:
You're right, because of timeout, lock may fail (TimeoutError exception).
I use timeout but i don't think about it the correct way.
I should correct that.

Reply:
I think timeout usage is correct. You have an except after that, it's ok.